spec/forms/response_spec.rb

```diff
@@ -2,38 +2,110 @@
 
 RSpec.describe Response, type: :model do
 
-  context 'when correctly populated' do
+  let(:populated_response) {
+    described_class.new(
+      defend_claim: true, defend_claim_facts: 'lorem ipsum defend claim facts'
+    )
+  }
+
+  let(:two_thousand_five_hundred_and_one_characters) {
```

Comment:
There is a gem called 'faker' that might be of interest for stuff like this

Reply:
Yeah I thought about it but this seemed a little quicker as I don't want anything other than long strings.